requirements.txt

```diff
@@ -1 +1,3 @@
-
+pyscreenshot==3.0
```

Comment:
May I ask which operating system both you @sabderemane and @Veershah26 you are running? Even after doing an install of those three libraries, when I run main.py the assistant errors out and I see this:
```
    raise AttributeError("Could not find PyAudio; check installation")
AttributeError: Could not find PyAudio; check installation
```
When I try to manually do a `pip install pyaudio` I get another error and I cannot procede. I am using a MacOS w/Intel processor.
```
Collecting PyAudio
  Using cached PyAudio-0.2.11.tar.gz (37 kB)
Building wheels for collected packages: PyAudio
  Building wheel for PyAudio (setup.py) ... error
  ERROR: Command errored out with exit status 1:...
```

Reply:
Which python version are you using ? I'm on same system (Big Sur)

Reply:
Also BigSur 11.6. I just found a fix looking at the pyAudio instructions here https://people.csail.mit.edu/hubert/pyaudio/
```
brew install portaudio
pip install pyaudio
```

Reply:
Nope i didn't have to when I've done the installation. I have to check which are necessary to install with brew